src/sql/workbench/contrib/scripting/browser/scriptingActions.ts

```diff
@@ -307,39 +307,28 @@ CommandsRegistry.registerCommand({
 // Refresh Action for Scriptable objects
 CommandsRegistry.registerCommand({
 	id: OE_REFRESH_COMMAND_ID,
-	handler: async (accessor, args: ObjectExplorerActionsContext) => {
+	handler: async (accessor, args: ObjectExplorerActionsContext): Promise<void> => {
 		const connectionManagementService = accessor.get(IConnectionManagementService);
 		const capabilitiesService = accessor.get(ICapabilitiesService);
 		const objectExplorerService = accessor.get(IObjectExplorerService);
-		const errorMessageService = accessor.get(IErrorMessageService);
+		const logService = accessor.get(ILogService);
 		const connection = new ConnectionProfile(capabilitiesService, args.connectionProfile);
-		let treeNode: TreeNode;
 		if (connectionManagementService.isConnected(undefined, connection)) {
-			treeNode = await getTreeNode(args, objectExplorerService);
-			if (treeNode === undefined) {
```

Comment:
I'm wondering why we specifically checked for `undefined`. Do we have any special behavior when node is `null`?

Reply:
It was probably just code that was being more specific than it needed to be. I don't know of a good reason why we'd do anything differently (I also don't think we can even get null back from this regardless - the OE service just returns undefined if it can't find the requested node)